src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -54,56 +54,64 @@
               No results found.
             </p>
           </div>
-          <Reference
-            v-for="(symbol, index) in filteredSymbols"
-            class="quick-navigation__reference"
-            :key="symbol.uid"
-            :url="symbol.path"
-            @click.native="closeQuickNavigationModal"
-            @focus.native="focusIndex(index)"
-          >
-            <div
-              class="quick-navigation__symbol-match"
-              ref="match"
-              role="list"
-              :class="{ 'selected' : index == focusedIndex }"
+          <div class="quick-navigation__refs">
```

Comment:
I think the `__refs` style should be applied only when results were found because it's messing with the `no-results` alignment. Rendering this div based on the value of `noResultsWereFound` can do the fix.
<img width="1432" alt="Screen Shot 2023-03-03 at 12 49 48 PM" src="https://user-images.githubusercontent.com/49292858/222791948-dd450e3e-bba7-4045-aa17-80d32a6cc546.png">

Reply:
Good catch! I'll try to fix the issue with the alignment.

Reply:
Fixed with 11ea6d44fdd2dd4badebd7ea2137443206c71121 :)